adal/src/main/java/com/microsoft/aad/adal/ADFSWebFingerValidator.java

```diff
@@ -0,0 +1,70 @@
+package com.microsoft.aad.adal;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * Validates trusts between authorities and ADFS instances using DRS metadata and WebFinger.
+ */
+final class ADFSWebFingerValidator {
+
+    private ADFSWebFingerValidator() {
+        // utility class
+    }
+
+    /**
+     * Used for logging.
+     */
+    private static final String TAG = "AdfsWebFingerValidator";
```

Comment:
Fixed.

If we care, this is handled inconsistently at various places in the code: can resolve in separate PR, in this one, or not at all if you're indifferent.

![screen shot 2017-01-03 at 1 18 11 pm](https://cloud.githubusercontent.com/assets/990063/21623467/5b80b2b4-d1b7-11e6-8313-32fdcc5c52db.png)